packages/terra-site/src/examples/dynamic-grid/ULayout.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import DynamicGrid from 'terra-dynamic-grid';
+
+import Card from './Card';
+
+const template = {
+  'grid-template-columns': '1fr 1fr 1fr',
+  'grid-template-rows': 'auto',
```

Comment:
IE/Edge are rendering the ULayout without a gap between rows but the other browsers render with a gap between rows. The grid-gap on line 9 should be applied to both columns and rows right?

## IE Screenshot
<img width="1267" alt="screen shot 2017-08-29 at 10 57 06 am" src="https://user-images.githubusercontent.com/633148/29831474-e7e82ec2-8caa-11e7-8d74-862c7085f014.png">

## Chrome Screenshot
<img width="1434" alt="screen shot 2017-08-29 at 11 16 58 am" src="https://user-images.githubusercontent.com/633148/29831732-a46a848c-8cab-11e7-9693-18fdb3ce6bbe.png">

Switching this to `auto 10px auto` or `auto 10px` seems to fix this in IE/Edge

## IE after switching grid-template-rows to `auto 10px auto`
<img width="1271" alt="screen shot 2017-08-29 at 11 14 18 am" src="https://user-images.githubusercontent.com/633148/29831644-656c3c94-8cab-11e7-85a0-9cd877f13f11.png">

## IE after switching grid-template-rows to `auto 10px`
<img width="1278" alt="screen shot 2017-08-29 at 11 23 23 am" src="https://user-images.githubusercontent.com/633148/29832076-a1240e1e-8cac-11e7-82f5-cf3e61484374.png">


Not sure if the 10px grid-gap on line 9 is being applied to rows via -ms properties to simulate gap.